$ yarn run jest -- factorial.test.js

 FAIL  ./factorial.tail.test.js
  ● Should avoid too much recursion

    expect(function).not.toThrowError(regexp)

    Expected the function not to throw an error matching:
      /stack size exceeded/
    Instead, it threw:
      RangeError: Maximum call stack size exceeded

  ✓ Should calculate the factorial of the given number (5ms)
  ✓ Should not accept negative values (1ms)
  ✕ Should avoid too much recursion (10ms)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 2 passed, 3 total
Snapshots:   0 total
Time:        0.656s, estimated 1s
Ran all test suites matching "factorial.test.js".
